traversal/planner/PlannerVertex.java

```diff
@@ -225,15 +228,15 @@ protected Properties.Thing newProperties() {
         }
 
         @Override
-        void updateObjective(GraphManager graph) {
+        void computeCost(GraphManager graphMgr) {
             if (props().hasIID()) {
-                setObjectiveCoefficient(1);
+                cost = 1;
             } else {
                 assert !props().types().isEmpty();
                 if (iterate(props().predicates()).anyMatch(p -> p.operator().equals(EQ))) {
-                    setObjectiveCoefficient(props().types().size());
+                    cost = props().types().size();
                 } else {
-                    setObjectiveCoefficient(graph.data().stats().thingVertexSum(props().types()));
+                    cost = graphMgr.data().stats().thingVertexSum(props().types());
```

Comment:
As in `PlannerEdge`, replace `setObjectiveCoefficient()` calls with directly setting `cost`.